tests/test_operation.py

```diff
@@ -1186,14 +1199,37 @@ def test_basis_state_decomposition(self, monkeypatch):
     def test_qubit_state_vector_decomposition(self, monkeypatch):
         """Test the decomposition of QubitStateVector calls the
         MottonenStatePreparation template"""
-        state = np.array([1/2, 1j/np.sqrt(2), 0, -1/2])
+        state = np.array([1 / 2, 1j / np.sqrt(2), 0, -1 / 2])
         wires = [0, 1]
         call_args = []
 
         # We have to patch MottonenStatePreparation where it is loaded
-        monkeypatch.setattr(qml.ops.qubit, "MottonenStatePreparation", lambda *args: call_args.append(args))
+        monkeypatch.setattr(
+            qml.ops.qubit, "MottonenStatePreparation", lambda *args: call_args.append(args)
+        )
         qml.QubitStateVector.decomposition(state, wires=wires)
 
         assert len(call_args) == 1
         assert np.array_equal(call_args[0][0], state)
         assert np.array_equal(call_args[0][1], wires)
+
+
+class TestChannel:
+    """Unit tests for the Channel class"""
```

Comment:
(Minor suggestion): a small test case could involve having another `DummyOp` definition with a trivial `_kraus_matrices` classmethod defined. `_kraus_matrices`  and `kraus_matrices` could then be checked in a simple way, making sure that parameters are being passed and were stored correctly